src/components/Home/SelectedFolderDetails.js

```diff
@@ -0,0 +1,168 @@
+//@flow
+import React, { useEffect, useState } from "react"
+
+import CircularProgress from "@material-ui/core/CircularProgress"
+import Grid from "@material-ui/core/Grid"
+import { makeStyles } from "@material-ui/core/styles"
+// import { useDispatch } from "react-redux"
+import { useLocation } from "react-router-dom"
+
+import SubmissionDetailTable from "components/Home/SubmissionDetailTable"
+import WizardStatusMessageHandler from "components/NewDraftWizard/WizardForms/WizardStatusMessageHandler"
+import { FolderSubmissionStatus } from "constants/wizardFolder"
+import { ObjectStatus } from "constants/wizardObject"
+import { WizardStatus } from "constants/wizardStatus"
+// import { deleteObjectFromFolder } from "features/wizardSubmissionFolderSlice"
+import draftAPIService from "services/draftAPI"
+import folderAPIService from "services/folderAPI"
+import objectAPIService from "services/objectAPI"
+
+const useStyles = makeStyles(theme => ({
+  tableGrid: {
+    margin: theme.spacing(1, 0),
+  },
+  circularProgress: {
+    margin: theme.spacing(10, "auto"),
+  },
+}))
+
+const SelectedFolderDetails = (): React$Element<typeof Grid> => {
+  const classes = useStyles()
+  // const dispatch = useDispatch()
+  const [isFetchingFolder, setFetchingFolder] = useState(true)
+  const [connError, setConnError] = useState(false)
+  const [responseError, setResponseError] = useState({})
+  const [errorPrefix, setErrorPrefix] = useState("")
+  const [selectedFolder, setSelectedFolder] = useState({
+    folderTitle: "",
+    allObjects: [],
+    published: "",
```

Comment:
```suggestion
     published: boolean
```

Reply:
ah, sorry. This is initial state, not data types, so it should be `false/true`

Reply:
Value should be a default value, not type. Will default to `false`

Reply:
Value should be a default value, not type. Will default to `false`

Reply:
yes